sdk/keyvault/Azure.Security.KeyVault.Administration/src/AccessControlClient.cs

```diff
@@ -0,0 +1,322 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using System.Threading;
+using System.Threading.Tasks;
+using Azure.Core;
+using Azure.Core.Pipeline;
+using Azure.Security.KeyVault;
+using Azure.Security.KeyVault.Administration.Models;
+
+namespace Azure.Security.KeyVault.Administration
+{
+    /// <summary>
+    /// The Client.
+    /// </summary>
+    public class AccessControlClient
```

Comment:
Mentioned in the API file, but this should be more specific to Key Vault. Archboard generally doesn't consider the namespace or package name to be qualifying enough.